scripts/release.sh

```diff
@@ -19,6 +19,20 @@ rm snforge_std/Scarb.toml.bak 2> /dev/null
 sed -i.bak "/\[package\]/,/version =/ s/version = \".*/version = \"${VERSION}\"/" crates/snforge-scarb-plugin/Scarb.toml
 rm crates/snforge-scarb-plugin/Scarb.toml.bak 2> /dev/null
 
+# start: Update cache test data
+VERSION_UNDERSCORED=$(echo "$VERSION" | tr '.' '_')
+
+DIRECTORY="crates/forge/tests/data/forking/.snfoundry_cache"
+OLD_FILE=$(ls "$DIRECTORY" | grep -E '_v[0-9_]+\.json')
```

Comment:
```suggestion
OLD_FILE=$(find "$DIRECTORY" -type f -regex '.*_v[0-9][0-9_]*\.json')
```
or something along those lines.

Generally it is not advisable to use ls in such applications (or in scripts generally, eg problem: you can't distinguish files vs directories).

My suggestion should work for both linux and bsd (mac) find, but please test it beforehand.

Btw what are possible file names? maybe the regex could be improved somehow?

Reply:
Script updated, it works well